pkg/capture/capture_manager.go

```diff
@@ -482,7 +485,7 @@ func (cm *Manager) Close(ctx context.Context, ifaces ...string) {
 	logger.With(
 		"elapsed", time.Since(t0).Round(time.Millisecond).String(),
 		"ifaces", ifaces,
-	).Debug("closed interfaces")
+	).Info("closed interfaces")
```

Comment:
Increased the log level for these rare emissions to improve debugging capabilities (I'd like to know when a change happened, not only on debug level).